doc/radosgw/upgrade_to_jewel.rst

```diff
@@ -0,0 +1,37 @@
+=============
+RGW upgrading to Jewel versions 10.2.0, 10.2.1, 10.2.2 and 10.2.3
+=============
+
+.. versionadded:: Jewel
+
+Upgrade of :term:`Ceph Object Gateway` to older versions of jewel (up to 10.2.3 included) caused issues. This document describes the needed recovery procedure.
+
+Mixed version of :term:`Ceph Object Gateway` is not supported
+
+Backup of old configuration
+================
+rados mkpool .rgw.root.backup
+rados cppool .rgw.root .rgw.root.backup
+
+Fix confgiuration after upgrade
+================
+Stop all :term:`Ceph Object Gateway` running in the cluster.
+
+Run the following commands:::
+
+  $ radosgw-admin zonegroup get --rgw-zonegroup=default | sed 's/"id":.*/"id": "default",/g' | sed 's/"master_zone.*/"master_zone":"default",/g' > default-zg.json
+
+  $ raodsgw-admin zone get --zone-id=default > default-zone.json
+
+  $ radosgw-admin realm create --rgw-realm=myrealm
+
+  $ radosgw-admin zonegroup set --rgw-zonegroup=default < default-zg.json
+
+  $ radosgw-admin zone set --rgw-zone=default < default-zone.json
+
+  $ radosgw-admin zonegroup default --rgw-zonegroup=default
+
+  $ radosgw-admin zone default --rgw-zone=default
```

Comment:
nit the `zonegroup set` and `zone set` commands will accept the `--default` flag, so you could avoid these last two steps

and am i correct that a `radosgw-admin period update --commit` isn't needed here because radosgw will fall back to the local zone/zonegroup objects?